2016/ideas-list-swc.md

```diff
@@ -29,6 +31,25 @@ The key features are:
 * if time allows, integration with [AMY][]   (workshop management tool for
   [Software Carpentry][]).
 
+<table>
+
+<tr>
+    <td>Intensity</td>
+    <td>Moderate</td>
+</tr>
+
+<tr>
+    <td>Involves</td>
+    <td>Python, Django, REST API, HTML, CSS</td>
+</tr>
+
+<tr>
+    <td>Mentors</td>
+    <td>[@pbanaszkiewicz], [@rgaiacs]</td>
```

Comment:
Try
1. manually writing the table (https://help.github.com/articles/organizing-information-with-tables/)
2. using lazy-links (`[@pbanaszkiewicz][]`, and at the of document `  [@pbanaszkiewicz]: https://github.com/pbanaszkiewicz/`)